### PR TITLE
add 2023-2024

### PR DESCRIPTION
**Pull request by nicholasloveday**

### CODE REVIEW

**Comment by nicholasloveday** [comment]
@mareecarroll can you please check that snakemake runs for you with this pull request?

I have updated it to also generate SVG files as well as PDFs

**Comment by nicholasloveday** [comment]
Oh, let me move the updated figs to the published results dir

**Comment by nicholasloveday** [comment]
Okay, this is hopefully ready @mareecarroll

File: Snakefile [hunk 1]
```diff
@@ -11,7 +11,9 @@ rule all:
 rule fig_2:
   conda: "workflow/envs/notebooks.yml"
   log: "logs/fig_2.log"
-  output: "results/figures/fig2.pdf"
+  output: 
+    "results/figures/fig2.pdf"
```
Comment (mareecarroll) [inline Snakefile:15]:
```suggestion
    "results/figures/fig2.pdf",
```

File: Snakefile [hunk 2]
```diff
@@ -27,18 +29,24 @@ rule fig_3:
...
+    "results/figures/fig_3_skill_score_lead_day.svg",
+    "results/figures/fig_3skill_score_colourbar.svg",
   notebook: "src/fig_3_skill_score_plot.ipynb"
 
 rule fig_4:
   conda: "workflow/envs/notebooks.yml"
   log: "logs/fig_4.log"
-  output: "results/figures/fig_4_isoreg.pdf"
+  output: 
+    "results/figures/fig_4_isoreg.pdf"
```
Comment (mareecarroll) [inline Snakefile:40]:
```suggestion
    "results/figures/fig_4_isoreg.pdf",
```

**Comment by mareecarroll** [comment]
Those additional outputs should also be added to the `inputs:` section of the `rule all`

**Comment by nicholasloveday** [comment]
> Those additional outputs should also be added to the `inputs:` section of the `rule all`

Thanks @mareecarroll . I have added them

**Pull request merged**